scripts/check_frontend_test_coverage.py

```diff
@@ -137,21 +137,22 @@
     'learner-view-info.directive.ts',
     'learner-view-rating.service.ts',
     'list-of-sets-of-translatable-html-content-ids-editor.component.ts',
-    'list-of-tabs-editor.directive.ts',
-    'list-of-unicode-string-editor.directive.ts',
-    'logic-error-category-editor.directive.ts',
-    'logic-question-editor.directive.ts',
-    'math-expression-content-editor.directive.ts',
+    'list-of-tabs-editor.component.ts',
+    'list-of-unicode-string-editor.component.ts',
+    'logic-error-category-editor.component.ts',
+    'logic-question-editor.component.ts',
+    'math-expression-content-editor.component.ts',
     'mathjax-bind.directive.ts',
+    'mathjax.directive.ts',
```

Comment:
This is a new file in the list? Can we cover it?

Reply:
Covrered!